judgels-backends/uriel/uriel-api/src/main/java/judgels/uriel/api/contest/submission/bundle/ContestantAnswerSummaryResponse.java

```diff
@@ -0,0 +1,22 @@
+package judgels.uriel.api.contest.submission.bundle;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import java.util.List;
+import java.util.Map;
+import judgels.jophiel.api.profile.BasicProfile;
+import judgels.sandalphon.api.submission.bundle.ItemSubmission;
+import judgels.uriel.api.contest.submission.ContestSubmissionConfig;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableContestantAnswerSummaryResponse.class)
+public interface ContestantAnswerSummaryResponse {
+    BasicProfile getProfile();
+    ContestSubmissionConfig getConfig();
+    Map<String, List<ItemSubmission>> getSubmissionsByProblemJid();
```

Comment:
Can we follow the convention used in everywhere else?

```
getSubmissionsMap()
getProblemAliasesMap()
getProblemNamesMap()
getItemNumbersMap()
```

Reply:
Will be changed to the new format